Include/internal/pycore_pylifecycle.h

```diff
@@ -110,8 +110,6 @@ PyAPI_FUNC(void) _PyErr_Print(PyThreadState *tstate);
 PyAPI_FUNC(void) _PyErr_Display(PyObject *file, PyObject *exception,
                                 PyObject *value, PyObject *tb);
 
-PyAPI_FUNC(void) _PyThreadState_DeleteCurrent(_PyRuntimeState *runtime);
```

Comment:
I don't understand why do you remove this definition: it's still used, no?

Reply:
I am now exposing PyThreadState_DeleteCurrent() instead of the internal  _PyThreadState_DeleteCurrent() I thought. what are you thinking?

Reply:
_PyThreadState_DeleteCurrent() has to stay, it's part of https://pythoncapi.readthedocs.io/runtime.html effort to avoid global variables.